aurora/src/main/java/digital/murl/aurora/Aurora.java

```diff
@@ -0,0 +1,57 @@
+package digital.murl.aurora;
+
+import digital.murl.aurora.agents.Agent;
+import digital.murl.aurora.agents.AgentManager;
+import digital.murl.aurora.effects.Effect;
+import digital.murl.aurora.effects.EffectRegistrar;
+import digital.murl.aurora.effects.CacheBehavior;
+import digital.murl.aurora.regions.*;
+import digital.murl.aurora.regions.distributors.RegionDistributor;
+
+import java.util.HashMap;
+
+@SuppressWarnings("unused")
+public class Aurora {
+    public static <T extends Effect> void registerEffect(String name, Class<T> effect, HashMap<String, digital.murl.aurora.effects.Action> actions, CacheBehavior cacheBehavior, HashMap<String, String> paramSchemas) throws Exception {
+        EffectRegistrar.addEffect(name, effect, actions, cacheBehavior, paramSchemas);
+    }
+
+    public static void registerAgent(String name, Agent agent, HashMap<String, digital.murl.aurora.agents.Action> actions, HashMap<String, String> paramSchemas) {
+        AgentManager.registerAgent(name, agent, actions, paramSchemas);
```

Comment:
i'm guessing paramSchemas is keyed by action name?

Reply:
yes, it is